src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27229.cs

```diff
@@ -1,4 +1,5 @@
-﻿using NUnit.Framework;
+#if TEST_FAILS_ON_WINDOWS
```

Comment:
Could we include a comment here to help in the future to re-enable it on Windows?